### PR TITLE
feat(next): Optimize the space information display

### LINKED ISSUE #656: Optimize the space information display
- [Display the number](https://www.figma.com/file/vqpglMGW8psHKB00eIVDUV/OpenSquare?node-id=8478%3A39372) of strategy and network
- If threshold amount is a large number, [use abbreviations](https://www.figma.com/file/vqpglMGW8psHKB00eIVDUV/OpenSquare?node-id=8511%3A47417)
- Display a maximum of one entry in strategy, replace the excess entries with an ellipsis
- Display a maximum of three icons in network, replace the excess icons with an ellipsis
- [When hover over the title](https://www.figma.com/file/vqpglMGW8psHKB00eIVDUV/OpenSquare?node-id=8478%3A50767)(threshold, strategy, network), add underline. (remove tooltip effect)
- [Click the title](https://www.figma.com/file/vqpglMGW8psHKB00eIVDUV/OpenSquare?node-id=8478%3A50767) to display the details popup

preview
image

### PR DESCRIPTION
**Pull request by 2nthony**
close #656 

- [x] [Display the number](https://www.figma.com/file/vqpglMGW8psHKB00eIVDUV/OpenSquare?node-id=8478%3A39372) of strategy and network
- [x] If threshold amount is a large number, [use abbreviations](https://www.figma.com/file/vqpglMGW8psHKB00eIVDUV/OpenSquare?node-id=8511%3A47417)
- [x] Display a maximum of one entry in strategy, replace the excess entries with an ellipsis
- [x] Display a maximum of three icons in network, replace the excess icons with an ellipsis
- [x] [When hover over the title](https://www.figma.com/file/vqpglMGW8psHKB00eIVDUV/OpenSquare?node-id=8478%3A50767)(threshold, strategy, network), add underline. (remove tooltip effect)
- [x] [Click the title](https://www.figma.com/file/vqpglMGW8psHKB00eIVDUV/OpenSquare?node-id=8478%3A50767) to display the details popup

### CODE REVIEW

File: packages/next/components/listInfo.js [hunk 1]
```diff
@@ -110,10 +110,12 @@ export default function ListInfo({ space }) {
           <AboutIcon src="/imgs/icons/threshold.svg" />
           <div>
             <AboutName>Threshold</AboutName>
-            <AboutDetail>{`${toPrecision(
-              space.proposeThreshold,
-              space.decimals
-            )} ${space.symbol}`}</AboutDetail>
+            <AboutDetail>
+              <ValueDisplay
```
Comment (YoshiyukiSakura) [inline packages/next/components/listInfo.js:137]: please use a closed tag insead of empty brace
`<ValueDispaly />`

File: packages/next/components/listInfo/details.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        <img src="/imgs/icons/info.svg" />
+      </DetailsWrapper>
+
+      <Divider />
+
+      <div>
+        <DetailsItem>
+          <div>Threshold</div>
+          <DetailsValue>
+            {toPrecision(space.proposeThreshold, space.decimals)} {space.symbol}
```
Comment (YoshiyukiSakura) [inline packages/next/components/listInfo/details.js:87]: please double check if we need to do a `abbreviateBigNumber` here

File: packages/next/components/listInfo/index.js [hunk 3]
```diff
@@ -101,59 +130,57 @@ export default function ListInfo({ space }) {
...
           <div>
-            <AboutName>Threshold</AboutName>
-            <AboutDetail>{`${toPrecision(
-              space.proposeThreshold,
-              space.decimals
-            )} ${space.symbol}`}</AboutDetail>
+            <AboutName onClick={handleShowModal}>
+              Strategy({strategyCount})
+            </AboutName>
+            <StrategyAboutDetail>
```
Comment (YoshiyukiSakura) [inline packages/next/components/listInfo/index.js:151]: please add a html native `title` prop here, so users can hover to see full content

Comment (YoshiyukiSakura) [reply packages/next/components/listInfo/index.js:151]: also, use a `span` tag here is perefred

File: packages/next/components/listInfo/details.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+export default function Details({ space }) {
+  const strategyCount = space.weightStrategy?.length || 0;
+  const networkCount = space.networks?.length || 0;
+
+  return (
+    <Wrapper>
+      <LogoWrapper>
+        <SpaceLogo spaceId={space.id} />
+        <LogoName>{space.name}</LogoName>
```
Comment (YoshiyukiSakura) [inline packages/next/components/listInfo/details.js:72]: css line-height of this should be 28px

File: packages/next/components/listInfo/details.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+      <DetailsWrapper>
+        <DetailsTitle>About</DetailsTitle>
+        <img src="/imgs/icons/info.svg" />
+      </DetailsWrapper>
+
+      <Divider />
+
+      <div>
+        <DetailsItem>
```
Comment (YoshiyukiSakura) [inline packages/next/components/listInfo/details.js:84]: css line-height of this should be 24px

File: packages/next/components/listInfo/details.js [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+  justify-content: space-between;
+  margin-bottom: 16px;
+`;
+
+const DetailsValue = styled.div`
+  ${p_14_medium};
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  margin-bottom: 4px;
```
Comment (YoshiyukiSakura) [inline packages/next/components/listInfo/details.js:57]: margin seems redundant

**Comment by YoshiyukiSakura** [comment]
image

**Comment by 2nthony** [comment]
> image

image

File: packages/next/components/listInfo/details.js (hunk 2 above)
Comment (2nthony) [reply packages/next/components/listInfo/details.js]: indeed, cuz `ValueDisplay` uses it.

File: packages/next/components/listInfo/details.js [hunk 7]
```diff
@@ -90,7 +90,10 @@ export default function Details({ space }) {
         <DetailsItem>
           <DetailsLabel>Threshold</DetailsLabel>
           <DetailsValue>
-            {toPrecision(space.proposeThreshold, space.decimals)} {space.symbol}
+            {abbreviateBigNumber(
```
Comment (YoshiyukiSakura) [inline packages/next/components/listInfo/details.js:93]: can we apply a <ValueDisplay> here?

Comment (2nthony) [reply packages/next/components/listInfo/details.js:93]: Not sure for now, the issue requires `(remove tooltip effect)`, I am asking the designer for details.

**Pull request merged**